src/hangman/SinglePlayer.java

```diff
@@ -0,0 +1,53 @@
+package hangman;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class SinglePlayer {
+
+	Tracker tracker;
+	Hangman hangman;
+
+	public void playSingle(Tracker tracker, Hangman hangman) {
+		
+		// Scanner to collect user input
+		Scanner scanner = new Scanner(System.in);
+		// a while loop
+		while (tracker.attemptsCount < 7 && tracker.lettersRemained > 0) {
+
+		
+			System.out.println("Enter a letter or a word: ");
+			String userEntry = scanner.nextLine().toUpperCase();
+
+			// check if userEntry contains any symbols other than letters
+			if (!userEntry.matches("[A-Z]+")) {
+				System.out.println("Please, enter valid input");
+			} else {
+
+				// check if user entry is a word or a letter
+				if (userEntry.length() > 1) {
+					// when the user entered a word
+					tracker.compareWord(userEntry);
```

Comment:
I like that this is in a method but does it make sense to be on the Tracker object or should it be part of the game logic?